### DISCUSSION TITLE
Bol şehirli soru

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#210] [Zeka Sorusu]
Bay Ankara, Bay İstanbul, Bay İzmir ve Bay Konya ; Ankara, İstanbul, İzmir ve Konyada doğmuş dört arkadaştır. Ancak hiç birinin soyadı doğduğu yere ait değildir. Bu dört arkadaşla ilgili şu önermeler yapılır.
Bay Ankara, Konyalıdır.
Bay İzmir, Ankaralıdır.
Bay İstanbul Konyalı değildir.
Bay Konya İstanbullu değildir.
Bu önermelerden sadece biri doğru olduğuna göre kim İzmirlidir.

**Comment by OmerAkbayrak** [comment]
### Bay Ankara İzmirlidir.
**1 ve 4. önerme doğru olamaz çünkü bu önermeler doğru olduğunda birden fazla önerme doğru olmak zorunda kalır. Bu durumda geriye 2 ve 3. önerme kalır, 2. önerme doğru olduğunda Bay Ankara İzmirli, Bay İzmir Ankaralı, Bay İstanbul Konyalı, Bay Konya İstanbullu olur.
3. önerme doğru olduğunda ise Bay Ankara İzmirli, Bay İzmir Konyalı, Bay İstanbul Ankaralı, Bay Konya İstanbullu olur. Ve böylece her durumda Bay Ankara İzmirli olur.**
